contracts/governance/AxelarMultisigMintLimiter.sol

```diff
@@ -0,0 +1,32 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.9;
+
+import { SafeNativeTransfer } from '@axelar-network/axelar-gmp-sdk-solidity/contracts/utils/SafeTransfer.sol';
+import { IMultisigExecute } from '../interfaces/IMultisigExecute.sol';
+import { MultisigBase } from '../auth/MultisigBase.sol';
+
+contract AxelarMultisigMintLimiter is MultisigBase, IMultisigExecute {
+    using SafeNativeTransfer for address;
+
+    constructor(address[] memory accounts, uint256 threshold) {
```

Comment:
Maybe the constructor should be present in MultisigBase instead to avoid duplication?